registration/templates/registration/password_reset_complete.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% trans "You may now" %} <a href="{{ login_url }}">{% trans "log in" %}</a>.
```

Comment:
My experience with translations in Django is limited, but after a quick refresher with the documentation I think this is a great approach.

I'd suggest making the change you suggested as a separate commit. I'll happily do a review and merge.